templates/instances/index.html

```diff
@@ -17,14 +17,23 @@ <h1 class="title is-1">{{ _("Join BookWyrm") }}</h1>
         </p>
     </div>
 
-    <div class="block columns is-multiline">
+    <div class="dropdown">
```

Comment:
This element doesn't work right with my screen reader (VoiceOver on mac). Visually, I can parse that the button is acting as a dropdown, but the screen reader doesn't get that info, and the elements aren't shown as clickable. There are a number of different ways to address this, but how would you feel about using a series of buttons rather than a dropdown? Something roughly like:
<img width="480" alt="Screen Shot 2022-06-28 at 8 12 56 AM" src="https://user-images.githubusercontent.com/1807695/176215862-ec3dc4a5-a19f-4ec9-bf0d-5eb5ad12044a.png">

Reply:
Interesting, maybe a lack of ARIA tags to help properly parse it? Regardless, there's a small enough amount of sorting options that I think that would work just fine

Reply:
ARIA tags would probably work, but broadly, I prefer to use native elements rather than ARIA overrides when possible.